packages/web-frontend/src/containers/UserMenu/index.js

```diff
@@ -146,125 +88,104 @@ const UserMenu = ({
   const primaryColor = customLandingPagePrimaryColor || DARK_BLUE;
   const secondaryColor = customLandingPageSecondaryColor || WHITE;
 
-  const signInText = isCustomLandingPage ? 'Log in' : 'Sign In / Register';
-
-  let menuItems = [];
+  const isDrawer = isMobile();
 
-  const logout = {
-    text: 'Logout',
-    action: onAttemptUserLogout,
-  };
-  const changePassword = {
-    text: 'Change Password',
-    action: onToggleChangePasswordPanel,
-  };
-  // if is a custom landing page, display only the appropriate items
-  if (isCustomLandingPage) {
-    const visitMainSite = {
-      text: 'Visit tupaia.org',
-      type: 'link',
-      url: window.location.origin,
-    };
-    menuItems = isUserLoggedIn
-      ? [
-          visitMainSite,
-          changePassword,
-          {
-            ...logout,
-            action: onAttemptLandingPageLogout,
-          },
-        ]
-      : [visitMainSite];
-  } else {
-    // otherwise display as usual
-    const viewProjects = {
-      text: 'View projects',
-      action: onOpenViewProjects,
-    };
-    const helpCentre = {
-      text: 'Help centre',
-      type: 'link',
-      url: 'https://beyond-essential.slab.com/posts/tupaia-instruction-manuals-05nke1dm',
-    };
-    menuItems = isUserLoggedIn
-      ? [
-          viewProjects,
-          changePassword,
-          {
-            text: 'Request country access',
-            action: onToggleRequestCountryAccessPanel,
-          },
-          helpCentre,
-          {
-            text: 'Logout',
-            action: onAttemptUserLogout,
-          },
-        ]
-      : [viewProjects, helpCentre];
-  }
+  // When is mobile, use a drawer menu, otherwise use a popover menu
+  const MenuComponent = isDrawer ? DrawerMenu : PopoverMenu;
 
   const toggleUserMenu = () => {
     setMenuOpen(!menuOpen);
   };
 
-  const closeUserMenu = () => {
+  const onCloseMenu = () => {
     setMenuOpen(false);
   };
 
-  const showRegisterButton = !isUserLoggedIn && isCustomLandingPage;
+  // Create the menu items
```

Comment:
```suggestion
  // Create the menu items
    const BaseMenuItem = props => <MenuItem onCloseMenu={onCloseMenu} {...props} />;
```

Probably worth making the base component to re-use since it's done a few times.